blogpost_10.html

```diff
@@ -0,0 +1,63 @@
+    <link rel="canonical" href="https://alexxcons.github.io/" />
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
+    <meta name="keywords" content="Xfce thunar GSoC alexxcons" />
+    <title>Alexxcon's Software Development Blog</title>
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+</head>
+
+<body>
+    <div id="wrapper">
+        <div id="header">
+            <h1><a href="index.html">Alexxcon's Software Development Blog</a></h1>
+        </div>
+        <div id="body" class="clear">
+            <h4>Apr 20,2024</h4>
+            <h2>Xfce is moving from IRC to Matrix</h2>
+
+            <p>After a 6-month trial period, as of April 1, 2024, <a href="https://www.xfce.org/">Xfce</a> moved its official communication channels from IRC to Matrix.</p>
+            <p>The old IRC channels will remain open, so you can still hang out there. However, the 'official' channels linked in the <a href="https://docs.xfce.org/contribute/dev/start#communication_with_the_xfce_development_team">Xfce Wiki</a> now will be the Matrix channels.</p>
+
+            <p>This change affects the following channels:</p>
+
+            <ul>
+                <li><a href="irc://irc.libera.chat/#xfce">#xfce</a> on libera.chat --&gt; <a href="https://matrix.to/#/#xfce:matrix.org">#xfce:matrix.org</a> -- general user support and discussion</li>
+                <li><a href="irc://irc.libera.chat/#xfce-dev">#xfce-dev</a> on libera.chat --&gt; <a href="https://matrix.to/#/#xfce-dev:matrix.org">#xfce-dev:matrix.org</a> -- development discussion</li>
+                <li><a href="irc://irc.libera.chat/#xfce-commits">#xfce-commits</a> on libera.chat --&gt; <a href="https://matrix.to/#/#xfce-commits:matrix.org">#xfce-commits:matrix.org</a> -- notable GitLab activity
+                </li>
+            </ul>
+
+            <p><b>Reasoning</b></p>
+            
+            <p>In the past, it was possible to connect and idle in libera IRC channels via <a href="https://matrix.org/ecosystem/bridges/irc/">Matrix.org bridge</a>.
+            Using Matrix to connect is very convenient for several reasons:</p>
+
+            <ul>
+                <li>When accessing IRC channels via Matrix bridge, it's not necessary to manage an <a href="https://en.wikipedia.org/wiki/BNC_(software)">IRC bouncer</a>.
+                    Bridged Matrix channels retain messages that arrive while you are offline.</li>
+                <li>Chat history is synchronised across multiple devices by default. No special BNC setup is required.</li>
+                <li>The entry barrier for Matrix is lower than for IRC.</li>
+                <li>On IRC, after answering a support question, you might realize that the person asking it went offline hours ago (no BNC used) and as such you wasted your time. Wont happen if the question was raised by a Matrix user.</li>
```

Comment:
Apologies for nitpicking, 😄 but I think you're missing an `'` in `Wont` (here and in both the feed files).

Reply:
Thank you for the nitpick, it's very welcome! Fixed now.